lms/static/js/spec/learner_dashboard/certificate_view_spec.js

```diff
@@ -0,0 +1,66 @@
+define([
+        'backbone',
+        'jquery',
+        'js/learner_dashboard/views/certificate_view'
+    ], function (Backbone, $, CertificateView) {
+        
+        'use strict';
+        /*jslint maxlen: 500 */
```

Comment:
What is this for? There should not be blanket overrides for a whole file, any necessary overrides should be targeted to the lines (ideally with an explanation of why jslint should ignore the error)